src/components/Autocomplete/components/MappedAction/MappedAction.tsx

```diff
@@ -0,0 +1,122 @@
+import React, {useMemo} from 'react';
```

Comment:
We created a new action component to map the difference between the old & new API